db/migrations/02_add_timestamp_to_peeps_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN time TIMESTAMP;
```

Comment:
I used separate date and time fields since I wasn't sure/in the mood to be figuring out how to parse what it would return. This is obviously more efficient from a storage perspective. Nice one.